src/services/k8s/knative.go

```diff
@@ -71,7 +74,63 @@ func loadManifest(project *project.Project, dockerImage docker.DockerImage) (*se
 	return service, nil
 }
 
-func Apply(namespace string, config *rest.Config, project *project.Project, dockerImage docker.DockerImage) error {
+func loadEnvFile(envFile string) ([]corev1.EnvVar, error) {
+	var envVarList []corev1.EnvVar
+	if _, err := os.Stat(envFile); err != nil {
+		if (os.IsNotExist(err)) && (path.Base(envFile) == ".env.initium") {
```

Comment:
Can you move `.env.initium` in the `default` package, like we do [here](https://github.com/nearform/initium-cli/pull/115/files#diff-6503267b158ca0aa967d389ea5c411abb8cf7a4e9764a6ff1bafb0ef0ac1f3baR135)? Thanks